src/app/components/home/assignment-chooser/assignment-chooser.component.ts

```diff
@@ -120,4 +130,85 @@ export class AssignmentChooserComponent implements OnInit {
     this.modalRef = this.modalService.open(this.importConfirmation);
     return this.modalRef.result;
   }
+
+  toggleExpandRow(row) {
+    this.table.rowDetail.toggleExpandRow(row);
+  }
+
+
+  toggleExpandGroup(group) {
+    this.table.groupHeader.toggleExpandGroup(group);
+  }
+
+  /**
+   * Concatenate questionsChecker(), startDateChecker(), endDateChecker() and getNumberOfRespository() be displayed in the template
+   * @param row 
+   * @returns String
+   */
+  detailDisplayer(row) : String {
+    var resultAsString = this.getNumberOfRespository(row) + "" + this.questionsChecker(row) + this.startDateChecker(row) + this.endDateChecker(row);
```

Comment:
If the functions return the type String, I think that using quotation marks is not necessary